### PR TITLE
New: Support documentation link and text in a report

### PR DESCRIPTION
**Pull request by sarvaje**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [ ] Added/Updated related tests.

### CODE REVIEW

**Comment by sarvaje** [comment]
I have added support to test the documentation links.

> Also, what do you think about making this a list of documentation links? I know we only need one right now, but it seems likely we'll have cases where we'll want to provide multiple links in the future.

I have just read this hehehe, I will work on that.

**Comment by sarvaje** [comment]
This is ready for review :)

**Comment by sarvaje** [comment]
@molant @johnemau will you have time to take a look to this PR? or can we just go ahead and merge it?

File: packages/utils-tests-helpers/src/hint-runner.ts [hunk 1]
```diff
@@ -199,25 +199,68 @@ const validateResults = (t: ExecutionContext<HintRunnerContext>, sources: Map<st
...
+            });
+        });
 
-                return position.column === location.column &&
-                    position.line === location.line &&
-                    (!('range' in report.position) || (
-                        position.endLine === location.endLine &&
-                        position.endColumn === location.endColumn));
+        if (filteredByDocumentation.length === 0) {
+            // TODO: Improve message.
```
Comment (molant) [inline packages/utils-tests-helpers/src/hint-runner.ts:228]: Should we improve it now or just remove this comment?

Comment (sarvaje) [reply packages/utils-tests-helpers/src/hint-runner.ts:228]: do you have any suggestion for the message? hehehe
If not I can remove the comment

Comment (molant) [reply packages/utils-tests-helpers/src/hint-runner.ts:228]: It's a test message so I'm fine with it as it is 😊

**Pull request merged**